<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>72</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:700; color:#2f394e;&quot;&gt;We Work&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>871</width>
      <height>541</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../image.qrc">:/new/bg/image/login_background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>311</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../image.qrc">:/new/bg/image/login_background2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="log_id_line">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>240</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文宋体</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
    font-size:13px;
    background:transparent;
    border:none;
    border-bottom:1px solid rgb(229, 229, 229);
}
 
QLineEdit:hover{
    border-bottom:1px solid rgb(40,159, 191);
}
 
QLineEdit:focus{
    border-bottom:1px solid rgb(18, 183, 245);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="log_pass_line">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>290</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文宋体</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
    font-size:13px;
    background:transparent;
    border:none;
    border-bottom:1px solid rgb(229, 229, 229);
}
 
QLineEdit:hover{
    border-bottom:1px solid rgb(40,159, 191);
}
 
QLineEdit:focus{
    border-bottom:1px solid rgb(18, 183, 245);
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 13pt &quot;华文琥珀&quot;;
    color: #FCFAF7;
    background-color: #A0D4BD;
    border-color: #BEE0D0;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FCFAF7;
    background-color: #63BE9D;
    border-color: #176F57;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FCFAF7;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #63BE9D, stop:1 #46B797);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FCFAF7;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
    </property>
    <property name="text">
     <string>登      录</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_icon">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>240</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../image.qrc">:/new/icon/image/account_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="account_icon">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../image.qrc">:/new/icon/image/password_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="minus_buton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../image.qrc">
      <normaloff>:/new/icon/image/shrink_icon.png</normaloff>:/new/icon/image/shrink_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_button">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../image.qrc">
      <normaloff>:/new/icon/image/quit_icon.png</normaloff>:/new/icon/image/quit_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="hint_1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>330</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>log_id_line</zorder>
   <zorder>log_pass_line</zorder>
   <zorder>log_button</zorder>
   <zorder>label</zorder>
   <zorder>account_icon</zorder>
   <zorder>password_icon</zorder>
   <zorder>minus_buton</zorder>
   <zorder>quit_button</zorder>
   <zorder>hint_1</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
